src/resonanceReconstruction/rmatrix/legacy/unresolved/SpinGroup/test/evaluate.test.hpp

```diff
@@ -0,0 +1,1116 @@
+        Resonance( 2.150000e+4 * electronVolt, 8.602800e+0 * electronVolt, 1.434100e-3 * rootElectronVolt, 1.150000e-2 * electronVolt, 0. * electronVolt, 0. * electronVolt ),
+        Resonance( 2.250000e+4 * electronVolt, 8.586700e+0 * electronVolt, 1.431400e-3 * rootElectronVolt, 1.150000e-2 * electronVolt, 0. * electronVolt, 0. * electronVolt ),
+        Resonance( 2.350000e+4 * electronVolt, 8.570500e+0 * electronVolt, 1.428700e-3 * rootElectronVolt, 1.150000e-2 * electronVolt, 0. * electronVolt, 0. * electronVolt ),
+        Resonance( 2.450000e+4 * electronVolt, 8.554400e+0 * electronVolt, 1.426000e-3 * rootElectronVolt, 1.150000e-2 * electronVolt, 0. * electronVolt, 0. * electronVolt ),
+        Resonance( 2.550000e+4 * electronVolt, 8.538200e+0 * electronVolt, 1.423300e-3 * rootElectronVolt, 1.150000e-2 * electronVolt, 0. * electronVolt, 0. * electronVolt ),
+        Resonance( 2.650000e+4 * electronVolt, 8.522200e+0 * electronVolt, 1.420600e-3 * rootElectronVolt, 1.150000e-2 * electronVolt, 0. * electronVolt, 0. * electronVolt ),
+        Resonance( 2.750000e+4 * electronVolt, 8.506200e+0 * electronVolt, 1.418000e-3 * rootElectronVolt, 1.150000e-2 * electronVolt, 0. * electronVolt, 0. * electronVolt ),
+        Resonance( 2.850000e+4 * electronVolt, 8.490100e+0 * electronVolt, 1.415300e-3 * rootElectronVolt, 1.150000e-2 * electronVolt, 0. * electronVolt, 0. * electronVolt ),
+        Resonance( 2.950000e+4 * electronVolt, 8.474100e+0 * electronVolt, 1.412600e-3 * rootElectronVolt, 1.150000e-2 * electronVolt, 0. * electronVolt, 0. * electronVolt ),
+        Resonance( 3.000000e+4 * electronVolt, 8.465900e+0 * electronVolt, 1.411300e-3 * rootElectronVolt, 1.150000e-2 * electronVolt, 0. * electronVolt, 0. * electronVolt )
+      },
+      { 1, 0, 0, 0 } );
+
+    // spin groups
+    SpinGroup group00( std::move( elastic00 ), std::move( table00 ) );
+    SpinGroup group10( std::move( elastic10 ), std::move( table10 ) );
+
+    ReactionID elas( "n,Pu239->n,Pu239" );
+    ReactionID capt( "n,Pu239->capture" );
+    ReactionID fiss( "n,Pu239->fission" );
+
+    THEN( "cross sections can be calculated for l=0" ) {
+
+      std::map< ReactionID, CrossSection > xs;
+      group00.evaluate( 2500. * electronVolt, xs );
+      CHECK( 3 == xs.size() );
+      CHECK( 0.19150687014446557 == Approx( xs[ elas ].value ) );
+      CHECK( 8.0066413975172654E-002 == Approx( xs[ capt ].value ) );
+      CHECK( 1.8803733373232319 == Approx( xs[ fiss ].value ) );
+      xs.clear();
+
+        group00.evaluate( 2550. * electronVolt, xs );
+      CHECK( 3 == xs.size() );
+      CHECK( 0.53240949386785641   == Approx( xs[ elas ].value ) );
+      CHECK( 0.22921106466840691   == Approx( xs[ capt ].value ) );
+      CHECK( 1.1815059631732758    == Approx( xs[ fiss ].value ) );
+      xs.clear();
+
+      group00.evaluate( 2650. * electronVolt, xs );
+      CHECK( 3 == xs.size() );
+      CHECK( 0.25213434053502726   == Approx( xs[ elas ].value ) );
```

Comment:
🙄 
```suggestion
      CHECK( 0.25213434063502726   == Approx( xs[ elas ].value ) );
```

Reply:
What is the change you suggest because I don't see it.

Reply:
I'm just playing around. I changed a `5` to a `6` because a `6` is obviously more correct.